src/dlsproto/client/DlsClient.d

```diff
@@ -744,6 +744,38 @@ public class DlsClient : IClient
         this.node_handshake = new NodeHandshake;
     }
 
+     /***************************************************************************
+
+        Constructor with support for only the neo protocols. Accepts only
+        `Neo.Config`, not `IClient.Config`, for applications that use the neo
+        protocol only.
+
+        Params:
+            epoll = EpollSelectDispatcher instance to use
+            neo_config = swarm.neo.client.mixins.ClientCore.Config instance.
+                (The Config class is designed to be read from an application's
+                config.ini file via ocean.util.config.ConfigFiller.)
+            conn_notifier = delegate which is called when a connection attempt
+                succeeds or fails (including when a connection is
+                re-established). Of type:
+                void delegate ( IPAddress node_address, Exception e )
```

Comment:
Hm, is this correct? I think it's a smart union notifier now.

Reply:
You're right, copy-pasta error.

Reply:
It's probably wrong all over the place ;)